pkg/controllers/status/work_status_controller.go

```diff
@@ -222,6 +222,11 @@ func (c *WorkStatusController) syncWorkStatus(key util.QueueKey) error {
 		return nil
 	}
 
+	// Work in a suspend state means it never dispatches updates to the member cluster.
+	if helper.IsWorkSuspendDispatching(workObject) {
```

Comment:
Hi @a7i If we add this judgment to this position, we also discard the state collection.

Reply:
good catch, updated.